code/datums/shuttles.dm

```diff
@@ -123,6 +124,35 @@
 /datum/map_template/shuttle/minidropship
 	shuttle_id = SHUTTLE_TADPOLE
 	name = "Tadpole Drop Shuttle"
+	suffix = "_standard"
+	description = "The plain and simple old Tadpole-03 model."
+	///shuttle switch console name
+	var/display_name = "Tadpole Standard Model"
+	var/image_url = "https://i.ibb.co/7vk4Q1h/2023-04-25-12-13-59-Window.png" // "shuttle previews are bad idea" ok heres an url instead
```

Comment:
lol what use UI assets

Reply:
Also maybe better to use preview_shuttle with a cooldown to stop spam or cache it somehow

Reply:
i asked you people
four distinct times
what to use

what the fuck is an UI asset

Reply:
gottem